source/features/forked-to.tsx

```diff
@@ -51,25 +52,144 @@ async function updateForks(forks: string[]): Promise<void> {
-				<span className="fork-flag rgh-forked">
-					forked to <a href={`/${fork}`}>{fork}</a>
+		/*
+		ORIGINAL HTML:
+			<form class="button_to" method="post" action="/sindresorhus/refined-github/fork">
+				<button type="submit" name="organization" value="organization" tabindex="0"
+					class="btn-link f5 text-bold my-2 tooltipped tooltipped-se tooltipped-align-left-1" aria-label="Will be created as organization/refined-github." title="@organization">
+					<div class="d-flex flex-items-center">
+						<img class="avatar avatar-small mr-2" src="https://avatars2.githubusercontent.com/u/50828444?s=60&amp;v=4" alt="@organization" width="30" height="30">
+						organization
+					</div>
+				</button>
+				<input type="hidden" name="authenticity_token" value="sidifhfhsnklfdhvskdlfhkgdhgk">
+			</form>
+
+		NEEDED HTML:
+			<form method="post" action={fork.parentNode.href}>
+				<button type="submit" name="organization" value="organization" className="select-menu-item width-full"
+					aria-label="Will be created as organization/refined-github." title="@organization">
+
+					<span className="select-menu-item-gravatar select-menu-item-icon">
+						<img src="https://avatars1.githubusercontent.com/u/50828444?s=40&amp;v=4" alt="@organization" width="20" height="20" />
+					</span>
+					<span className="select-menu-item-text">
+						organization
+					</span>
+				</button>
+				<input type="hidden" name="authenticity_token" value="sidifhfhsnklfdhvskdlfhkgdhgk" />
+			</form>
+		*/
+	});
+	console.log(toBeForked);
+
+	for (const element of select.all('a.d-block', forkDialog)) {
+		const fork = element.cloneNode(true) as HTMLElement;
+		fork.className = 'select-menu-item';
+		fork.firstElementChild!.className = 'select-menu-item-gravatar select-menu-item-icon';
+		fork.firstElementChild!.after(
+			<span className="select-menu-item-text">
+				{fork.firstElementChild!!.nextSibling!}
+			</span>
+		);
+		toBeForked.push(fork as HTMLElement);
+		/*
+		ORIGINAL HTML:
+			<a class="d-block" href="/organization">
+				<img class="avatar" src="https://avatars3.githubusercontent.com/u/5607437?s=40&amp;v=4" alt="@organization" width="20" height="20">
+				organization
+			</a>
+
+		NEEDED HTML:
+			<a href="/organization"
+			   className="select-menu-item">
+				<span className="select-menu-item-gravatar select-menu-item-icon">
+					<img src="https://avatars1.githubusercontent.com/u/50828444?s=40&amp;v=4" alt="@organization" width="20" height="20" />
+				</span>
+				<span className="select-menu-item-text">
+					organization
 				</span>
-			);
-		}
+			</a>
+		*/
 	}
+	if (toBeForked.length > 0) {
+		detailsDialog.append(
+			<div className="select-menu-header">
+				<a className="select-menu-title" href={`/${getRepoURL()}/fork`}>Create fork</a>
+			</div>
+		);
+		detailsDialog.append(...toBeForked);
+	}
+
+
+	const forkButton = select('summary[title^="Fork your own copy of"]')!;
+	forkButton.after(detailsDialog);
+
+	forkDialog!.remove();
```

Comment:
This function does a lot, maybe we can't afford to maintain all this code for this feature. We should find shortcuts.

This shorter code gets us 80% there:


```js
// Code written in console
const dialog = $('[src$="/fork?fragment=1"]');
const menu = document.createElement('details-menu');

// Copies the attributes from `dialog` and from the `watch` `details-menu` item
[...dialog.attributes, ...$('.select-menu-modal').attributes].forEach(attribute => menu.setAttribute(attribute.name, attribute.value));

// Moves the content
menu.append(...dialog.childNodes);
dialog.replaceWith(menu);

// Final style fixes
menu.parentElement.classList.remove('details-overlay-dark');
menu.classList.add('right-0');

menu.querySelector('include-fragment').addEventListener('load', () => {
	menu.querySelectorAll('.p-4').forEach(el => el.classList.replace('p-4', 'p-2'));
	menu.querySelectorAll('.text-normal').forEach(el => el.remove());
})
```

Reply:
I much rather go back to [design 3](https://github.com/sindresorhus/refined-github/pull/2230#issuecomment-511156020) and for 2 or more forks let the "Create fork" menuitem go to the [fork page](https://github.com/sindresorhus/refined-github/fork).

Reply:
> I much rather go back to [design 3](https://github.com/sindresorhus/refined-github/pull/2230#issuecomment-511156020)

Design 3 is only better than 1 because of this comment on 1:

> I just realized that this is the same exact content that the regular Fork button, minus the chance to create more forks. It doesn't make much sense to have two buttons that do the same thing.

All that design 3 does, compared to 1, is move the "regular fork button" to the dropdown... only to go back to the same comment (2 buttons, similar comment) + adding an extra step to create forks.

---

So if my last suggestion isn't liked, I'd go to the first commit https://github.com/sindresorhus/refined-github/pull/2230/commits/706d2760fc7be3ff8dfc866067a74366e1078c5b

Reply:
I completely agree about the maintainability of [design 4](https://github.com/sindresorhus/refined-github/pull/2230#issuecomment-520178324); even creating it was a lot of hassle. 
Using a [lesser design](https://github.com/sindresorhus/refined-github/pull/2230#discussion_r313044618) to just get the dropdown menu and one step less, is not satisfying.

I'm more then happy to go back to [design 1](https://github.com/sindresorhus/refined-github/pull/2230#issue-297197052); no extra steps, easy visible to see the amount of forks, easy access to your fork(s) and maintainable code.

Reply:
# 🙌